TI-reports/2025/2025-Q1-GCP-WG.md

```diff
@@ -0,0 +1,45 @@
+
+We have two working group co-leads: [Daniel Appelquist | Samsung](https://github.com/torgo) and [Mike Bursell | Confidential Compute Consortium](https://github.com/MikeCamel) with support from [Crob](https://github.com/SecurityCRob), [Jeff Diecks](https://github.com/GeauxJD) and [Fukami](https://github.com/fukami) from OpenSSF staff.
+
+We are also exploring folding in the "CRA tech bi-weekly" call into the remit of this group.
+
+Since forming, we have spent a fair amount of time refining the charter, in particular the wording around how we open up the group as much as possible while also making it clear who gets to vote when and if a vote is required.
+
+We now have a schedule of calls for our SIGs and have started to take minutes in our main minutes doc. We are operating in a similar mode to the Best Practices Working group, with our SIGs reporting into the main working group call. 
+
+We held a special session at LF Member Summit which is also minuted in our minutes doc.
+
+## Awareness SIG
+
+The awareness SIG is the furthest along. It's led by Megan Knight from Arm. The scope is activities that drive awareness of the work of this group and of the regulatory landscape in general. The SIG has been marshalling blog posts, upcoming conference schedule, as well as the CRA introductory course.
+
+## Tooling SIG
+
+The Tooling SIG has is still in start-up mode. However we have one lead, [Puerco](https://github.com/puerco), and we're looking for a co-lead. The scope is to coordinate work on relevant tooling and processes. What tools already exist out there that can help maintainers, stewards and manufacturers? What additional features do we need from existing tools?
+
+## Standards SIG
+
+Standards workstream is still getting up to speed. One area of focus will be promoting Baseline as a potential "standard" that can help the ecosystem comply with the CRA.  CRob+Jory+Wheeler will be presenting an opinioned list of possible specs within the OpenSSF and LF to put forth for consideration for ETSI & CEN/CENALC during the 1April2025 call.
+
+## Questions/Issues for the TAC
+
+[none at this time?]
+
+## Additional Information
+
```

Comment:
```suggestion

Security Insights spec recently [incorporated](https://github.com/ossf/security-insights-spec/pull/117#pullrequestreview-2728878937) `steward` as a field in their schema. This is in response to an [issue](https://github.com/ossf/security-insights-spec/issues/106) we raised about how a project can specify its steward, which in turn came out of a discussion in the working group about whether we need a `steward.md`. file. So this might be one way - maybe our recommended way? - for a project to indicate its steward.

```